docs/v2-system-addon/data_events.md

```diff
@@ -539,3 +539,20 @@ This reports the user's interaction with Activity Stream Router.
   "event": ["CLICK_BUTTION" | "BLOCK"]
 }
 ```
+
+### Targeting error pings
+
+This reports when an error has occurred when parsing/evaluating a JEXL targeting string in a message.
+
+```js
+{
+  "client_id": "n/a",
+  "action": "asrouter_undesired_event",
```

Comment:
Does this look OK @ncloudioj ? Or should I use some kind of asrouter user event action?

Reply:
Yep, this is fine. Or you can make it even more specific, such as `onboarding_undesired_event`.